src/app/course/course-practice-page/course-practice-page.component.scss

```diff
@@ -0,0 +1,69 @@
+@import '~@taiga-ui/core/styles/taiga-ui-local';
+@import 'mixins';
+
+.flex-container {
+    margin-top: 3rem;
+    margin-bottom: 3rem;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    flex-direction: row;
+    position: relative;
+}
+
+.practice-background {
+    width: calc(100% + 4.5rem);
+    height: calc(4 * var(--tui-padding-l));
+    margin-top: calc(-1 * var(--tui-padding-l));
+    margin-left: calc(-2 * var(--tui-padding-l));
+    @include tui-space(bottom, 3);
+}
+
+.practice-question {
+    background-color: var(--tui-accent-hover);
+}
+
+.view-goals {
+    background-color: var(--tui-support-21);
+}
+
+.practice-tile {
+    border: 0.2rem solid #000000;
```

Comment:
Border with the `rem` unit is wrong. If the user zooms or has a larger font size the border gets bigger, which doesn't make sense.